src/ucp/core/ucp_ep.h

```diff
@@ -118,10 +118,14 @@ typedef struct ucp_ep_config {
     struct {
         /* Maximal total size of rndv_get_zcopy */
         size_t                 max_get_zcopy;
-        /* Threshold for switching from eager to rendezvous */
-        size_t                 thresh;
+        /* Threshold for switching from eager to RMA based rendezvous */
+        size_t                 rma_thresh;
+        /* Threshold for switching from eager to AM based rendezvous */
+        size_t                 am_thresh;
         /* threshold for switching from eager-sync to rendezvous */
-        size_t                 sync_thresh;
+        size_t                 sync_rma_thresh;
+        /* threshold for switching from eager-sync to rendezvous */
+        size_t                 sync_am_thresh;
```

Comment:
currently the thresholds for 'sync' are the same, before we incorporate the additional latency of sync ack into eager latency forumla